theme/icons.go

```diff
@@ -569,13 +569,16 @@ func (t *builtinTheme) Icon(n fyne.ThemeIconName) fyne.Resource {
 	return icons[n]
 }
 
-// ThemedResource is a resource wrapper that will return a version of the resource with the main color changed
-// for the currently selected theme.
+// ThemedResource is a resource wrapper used to represent a version of a resource.
```

Comment:
We should not remove the main information from the first sentence - this is all that will be shown in some IDEs

Reply:
Hmm, the words `main color` was the part I found misleading; like `return a version of the resource`.

I Think I picked the wrong word represent, rather than `used to alter the presentation of a resource`. but we're line length limited here right?

Reply:
Maybe it was perfect. IDK...

Reply:
Line length limited is not the same as sentence length limited.
You could put some words on line 2 then a "." part way along - IIRC.